packages/terra-clinical-header/src/Header.jsx

```diff
@@ -8,6 +8,11 @@ const cx = classNames.bind(styles);
 
 const propTypes = {
   /**
+   * Child element to be displayed on the right end of the header.
```

Comment:
There should be something in the prop descriptions to differentiate between when children should be used and when startContent should be used.